.github/stale.yml

```diff
@@ -32,6 +32,10 @@ markComment: >
   Thank you for your contribution!
   /cc @hanzei @jasonblais
 
+# Comment to post when removing the stale label.
+unmarkComment: >
+  Please remove the stale label
```

Comment:
Thanks Carlos!

@hanzei Thoughts on making this comment `/active` --> brief and uses a slash command to avoid mixing with regular comments.

Reply:
Sounds good